app/components/publications/batch-documents-publication-modal.hbs

```diff
@@ -0,0 +1,49 @@
+<WebComponents::AuModal>
+  <WebComponents::AuModal::Header @title={{ t "documents" }} @closeModal={{this.onCancel}} />
+  <WebComponents::AuModal::Body>
+    {{!-- template-lint-disable table-groups --}}
+    <table class="auk-table auk-table--styled auk-table--bordered">
+      <thead>
+        <tr>
+          <th>{{t "name"}}</th>
+          <th>{{t "document-type"}}</th>
+          <th></th>
+        </tr>
+      </thead>
+
+      <Documents::DocumentList
+        @pieces={{this.pieces}}
+        {{!-- should be tbody tag; workaround for Documents::DocumentList rendering a div --}}
+        style="display: table-row-group;"
+        as |pieces|>
+        {{#each pieces as |piece|}}
+          <tr>
+            <td>
+              {{piece.name}}
+            </td>
+            <td>
+              {{piece.file.extension}}
+            </td>
+            <td>
+              <WebComponents::AuButtonToolbar>
+                <WebComponents::AuButtonLink
```

Comment:
Ik heb hier gekozen voor links. Dat lijkt me het gebruiksvriendelijkst. Het nadeel is dat deze links iets dichter bij elkaar staan dan gewone toolbar-buttons.